assets/tests.py

```diff
@@ -0,0 +1,145 @@
+import json
+from automationcommon.tests.utils import UnitTestCase
+from rest_framework.test import APIClient
+
+
+class APIViewsTests(UnitTestCase):
+    def setUp(self):
+        self.maxDiff = None
+
+    def assertDictListEqual(self, d1, d2, msg=None):
+        """Compares two dictionary with lists (order doesn't matter)"""
+        for k, v in d1.items():
+            if v.__class__ == list:
+                d1[k] = set(v)
+        for k, v in d2.items():
+            if v.__class__ == list:
+                d2[k] = set(v)
+        return self.assertDictEqual(d1, d2, msg)
+
+    def test_asset_post_is_complete(self):
+        client = APIClient()
+        asset_dict = {
+            "data_subject": [
+                "students"
+            ],
+            "data_category": [
+                "research"
+            ],
+            "risk_type": [
+                "operational", "reputational"
+            ],
+            "storage_format": [
+                "digital", "paper"
+            ],
+            "paper_storage_security": [
+                "locked_cabinet"
+            ],
+            "digital_storage_security": [
+                "acl"
+            ],
+            "name": "asset1",
+            "department": "department test",
+            "purpose": "don't know",
+            "research": True,
+            "owner": "amc203",
+            "private": True,
+            "personal_data": False,
+            "recipients_category": "no idea",
+            "recipients_outside_eea": "",
+            "retention": "<=1",
+            "retention_other": "",
+            "storage_location": "Who knows"
+        }
```

Comment:
Could we perhaps replace these three big dicts with a ``GOOD_ASSET`` constant on the class and simply ``del`` or modify fields as required?